uploadNewMarcxmlRecords.py

```diff
@@ -46,10 +50,22 @@
         print(oclcNum + '.xml', iaId + '_marc.xml')
         shutil.copy(replacementDirectory + oclcNum + '.xml',
                     replacementDirectory + iaId + '_marc.xml')
-        r2 = item.upload(replacementDirectory + iaId + '_marc.xml')
+        key = 'force-update'
+        value = 'true'
+        updateDict = {key: value}
+        forceUpdate = item.modify_metadata(dict(updateDict))
+        removeDesc = item.modify_metadata(dict(description='REMOVE_TAG'))
+        removeSubj = item.modify_metadata(dict(subject='REMOVE_TAG'))
+        uploadMarc = item.upload(replacementDirectory + iaId + '_marc.xml')
         utc = datetime.datetime.utcnow()
         utcTime = utc.strftime('%Y-%m-%dT%H:%M:%SZ')
-        f.writerow([iaId] + [oclcNum] + [r2] + [utcTime])
+        f.writerow([iaId] + [oclcNum] + [uploadMarc] + [utcTime])
         rowCount -= 1
         print('Items remaining: ', rowCount)
-        print(r2)
+        print(uploadMarc)
+
+# print script run time
+elapsedTime = time.time() - startTime
+m, s = divmod(elapsedTime, 60)
+h, m = divmod(m, 60)
+print('Total script run time: ', '%d:%02d:%02d' % (h, m, s))
```

Comment:
Not a required change but this can be made easier and more readable by using a `datetime.timedelta` object. Calling the `__str__` method on it will print the format you want:

```python
td = datetime.timedelta(seconds=time.time() - startTime)
print("Elapsed time: {}".format(td))
```

Reply:
Excellent! I have carried that particular code block over from an early StackOverflow answer I found when learning Python and this is much more clean. Thank you so much!